MUXSDKStats/MUXSDKStatsTests/Utils/MUXSDKCore+Mock.m

```diff
@@ -0,0 +1,61 @@
+//
+//  MUXSDKCore+Mock.m
+//  MUXSDKStatsTests
+//
+//  Created by Nidhi Kulkarni on 2/4/20.
+//  Copyright © 2020 Mux, Inc. All rights reserved.
+//
+
+#import "MUXSDKCore+Mock.h"
+#include <objc/runtime.h>
+
+
+@implementation MUXSDKCore (Mock)
+
+static NSMutableDictionary *events;
+
++ (void) swizzleDispatchEvents {
```

Comment:
Nice job with this. It makes it much easier for us to write tests and I can see this was the only way you were able to test the behavior that you fixed.